engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/pass/analyse/TailCallTest.scala

```diff
@@ -238,9 +239,9 @@ class TailCallTest extends MiniPassTest {
           fnBody
             .asInstanceOf[Expression.Block]
             .returnValue
-            .asInstanceOf[Application.Prefix]
-            .arguments(2)
-            .value
+            .asInstanceOf[IfThenElse]
```

Comment:
`IfThenElse` as a builtin IR is indeed better than representing it as `Application.Prefix` with 2 arguments